docs/sources/reference/commandline/cli.md

```diff
@@ -48,6 +48,34 @@ These Go environment variables are case-insensitive. See the
 [Go specification](http://golang.org/pkg/net/http/) for details on these
 variables.
 
+## Configuration Files
+The Docker command line stores its configuration files in a directory
```

Comment:
@moxiegirl ok - picked up your edits with just a minor tweak to the start of the 2nd paragraph. See if you're ok with that.